rescue/2025-09-16/cosmic-helix/helix-renderer.mjs

```
@@ -0,0 +1,497 @@
+  Rationale:
+    - no motion or flashing; everything renders once on load
+    - muted palette keeps contrast gentle for ND-safe viewing
+    - numerology constants wire geometry to 3/7/9/11/22/33/99/144
+    - No motion or flashing; everything rendered once.
+    - Muted palette for calm contrast.
+    - Numerology constants wire geometry to 3, 7, 9, 11, 22, 33, 99, 144.
+*/
+
+export function renderHelix(ctx, opts) {
+  const { width, height, palette, NUM: N, notice } = opts;
+  const { width, height, palette, NUM: N } = opts;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'width'. Consider to delete it or rename it.

'width' is defined here:


(lint/suspicious/noRedeclare)